resources/test.toml

```diff
@@ -15,7 +15,7 @@ sqlite_path = ":memory:"
 
 [server]
   PORT = 8888  # will be lower cased when reading.
-  base_url = "https://localhost"
+  base_url = "http://localhost"
```

Comment:
Why do we need https? It will not work for the test server anyway without certificates?